docs/sources/dashboards/time-range-controls.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Time range controls"
+keywords = ["grafana", "dashboard", "documentation", "time range"]
+type = "docs"
+[menu.docs]
+aliases = ["/docs/grafana/latest/reference/timerange/"]
+name = "Time range controls"
+parent = "dashboards"
+weight = 7
++++
+
+# Time range controls
+
+Grafana provides several ways to manage the time ranges of the data being visualized, both at the dashboard level and the panel level. 
+
+This page describes supported time units and relative ranges, the common time controls, dashboard-wide time settings, and panel-specific time settings.
+
+## Time units and relative ranges
+
+The following time units are supported: `s (seconds)`, `m (minutes)`, `h (hours)`, `d (days)`, `w (weeks)`, `M (months)`, and `y (years)`. 
+
+The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/$unit` to the end.
+
+Here are some examples:
+
+Example relative range | From: | To:
+-------------- | ----- | ---
+Last 5 minutes | `now-5m` | `now`
+The day so far | `now/d` | `now`
+This week | `now/w` | `now/w`
+Week to date | `now/w` | `now`
+Previous Month | `now-1M/M` | `now-1M/M`
+
+## Common time range controls
+
+The dashboard and panel time controls have a common user interface (UI).
+
+<img class="no-shadow" src="/img/docs/time-range-controls/common-time-controls-7-0.png" max-width="700px">
+
+The options are defined below.
+
+### Current time range
+
+The current time range, also called the _time picker_, shows the time range currently displayed in the dashboard or panel you are viewing.
+
+Hover your curser over the field to see the exact time stamps in the range and their source (such as the local browser).
+
+<img class="no-shadow" src="/img/docs/time-range-controls/time-picker-7-0.png" max-width="300px">
+
+To change the current time range, click the down arrow next to the current time range. You can change the current time using a _relative time range_, such as the last 15 minutes, or an _absolute time range_, such as `now-45h`.
```

Comment:
The time range plus down arrow is "clickable". Also the absolute time range fields shows the relative time selected (if a relative time is selected), but an absolute time range is for example `2020-05-14 00:00:00 to 2020-05-15 23:59:59`. Not sure how to easily explain this 

```suggestion
Click on the current time range to change the time range. You can change the current time using a _relative time range_, such as the last 15 minutes, or an _absolute time range_, such as `2020-05-14 00:00:00 to 2020-05-15 23:59:59`.
```